BridgeBidder/Constraints/Losers.cs

```diff
@@ -59,7 +59,7 @@ public override bool Conforms(Call call, PositionState ps, HandSummary hs)
         }
     }
 
-    public class ShowsLosers : HasLosers, IShowsState
+    public class ShowsLosers : HasLosers, IShowsState, IDescribeConstraint
```

Comment:
I realized later that this one isn't called anywhere yet, but probably still good to have the description ready.